public/js/account.js

```diff
@@ -435,6 +435,16 @@ $(document).ready(function() {
                                     },
                                     success: function() {
                                         alert('Password successful changed.');
+                                        $('#form-old-password').val('');
+                                        $('#form-password').val('');
+                                        $('#form-confirm-password').val('');
+                                        $('#form-first-name').prop('disabled', false);
+                                        $('#form-last-name').prop('disabled', false);
```

Comment:
it doesn't matter either way, I think using .hide() and .show() is a little easier and clearer so I recommend we use that.  I changed the line to use .hide()